.vscode/launch.json

```diff
@@ -0,0 +1,45 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "java",
+            "name": "S3D Local",
+            "request": "launch",
+            "mainClass": "us.dot.its.jpo.ode.aws.depositor.AwsDepositor",
+            "projectName": "jpo-aws-depositor",
+            "args": ["--bootstrap-server", "<kafka host>:9092",
```

Comment:
Are these "< >" values just stand-ins to be replaced by the developer?

Reply:
If we're going to include the launch.json, let's document how to run it in the README. IE we're assuming the user has access to a kafka host as well as access/secret and appropriate keys. Is there a way to read those from a .env file maybe? I'm worried with having them forced to be replaced in this launch.json that someone will check in their configuration by accident and expose their keys.

Reply:
I've added a section on launch configurations to the README.

That's fair. I think I initially tried to use environment variables, but I recall running into trouble. I'll do some testing to see if I can get it to work with environment variables.

Reply:
VSCode doesn't seem to let you set environment variables from environment variables (method shown below), and I don't think it's possible to load them in from a .env file.

```
"env": {
    "AWS_ACCESS_KEY_ID": "${TEST}",
    "AWS_SECRET_KEY": "${TEST}"
}
```